slices/libtiff6.yaml

```diff
@@ -0,0 +1,16 @@
+package: libtiff6
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libdeflate0_libs
+      - libjbig0_libs
+      - libjpeg8_libs
+      - liblerc4_libs
```

Comment:
Same here - this doesn't exist for s390x and i386. And thus is not needed by `libtiff6` for those archs. Do you know if it is needed at all? Unfortunately Chisel doesn't support, yet, conditional dependencies, so we either 1) drop this dependency if `libtiff6` can work without it (even if with limited functionality), or 2) keep it, ack that it will fail for certain archs, but we leave a comment noting that

Reply:
Resolved separately